refinery/core/models.py

```diff
@@ -762,8 +774,18 @@ def get_pre_isa_archive(self):
                 AttributeError) as e:
             logger.debug("Couldn't fetch FileStoreItem: %s", e)
 
-    def share(self, group, readonly=True):
+    def share(self, group, readonly=True, readmetaonly=False):
```

Comment:
Doesn't really matter, but shouldn't parameters get underscores like other variables?

Reply:
@scottx611x Thoughts?

Reply:
I'd prefer to see: `read_meta_only=False`